src/components/SecondaryPanes/Breakpoints/index.js

```diff
@@ -81,41 +81,45 @@ class Breakpoints extends Component<Props> {
       ...breakpointSources.map(({ source, breakpoints }) => source)
     ];
 
-    return [
-      ...breakpointSources.map(({ source, breakpoints, i }) => {
-        const path = getDisplayPath(source, sources);
-        const sortedBreakpoints = sortSelectedBreakpoints(
-          breakpoints,
-          selectedSource
-        );
-
-        return [
-          <BreakpointHeading
-            source={source}
-            sources={sources}
-            path={path}
-            key={source.url}
-          />,
-          ...sortedBreakpoints.map(breakpoint => (
-            <Breakpoint
-              breakpoint={breakpoint}
-              source={source}
-              selectedSource={selectedSource}
-              key={makeBreakpointId(
-                getSelectedLocation(breakpoint, selectedSource)
-              )}
-            />
-          ))
-        ];
-      })
-    ];
+    return (
+      <div className="pane breakpoints-list">
+        {[
+          ...breakpointSources.map(({ source, breakpoints, i }) => {
+            const path = getDisplayPath(source, sources);
+            const sortedBreakpoints = sortSelectedBreakpoints(
+              breakpoints,
+              selectedSource
+            );
+
+            return [
+              <BreakpointHeading
+                source={source}
+                sources={sources}
+                path={path}
+                key={source.url}
+              />,
+              ...sortedBreakpoints.map(breakpoint => (
+                <Breakpoint
+                  breakpoint={breakpoint}
+                  source={source}
+                  selectedSource={selectedSource}
+                  key={makeBreakpointId(
+                    getSelectedLocation(breakpoint, selectedSource)
+                  )}
+                />
+              ))
+            ];
+          })
+        ]}
+      </div>
+    );
   }
 
   render() {
     return (
-      <div className="pane breakpoints-list">
+      <div>
         {this.renderExceptionsOptions()}
-        {this.renderBreakpoints()}
+        {!!this.props.breakpointSources.length && this.renderBreakpoints()}
```

Comment:
Nitpick: my preferred style would be to have this check in `renderBreakpoints`.

```diff
   renderBreakpoints() {
     const { breakpointSources, selectedSource } = this.props;
+    if (!breakpointSources.length) return null;
```

We could also remove the `empty` conditional class in `renderExceptionsOptions` and move this style to the breakpoints list instead (using margin-top + border-top + padding-top):

```css
.breakpoints-exceptions-options:not(.empty) {
  border-bottom: 1px solid var(--theme-splitter-color);
  margin-bottom: 3px;
}
```

Reply:
Addressed this 👍